src/components/SearchBar/SearchBar.vue

```diff
@@ -57,39 +43,43 @@ export default defineComponent({
         isActive: false,
       },
     ]);
+    const shorten = async (val) => {
+      if (val.length > 5) {
```

Comment:
I think that this doesn't have sense because API gives us different type of errors 
```
1 | No URL specified ("url" parameter is empty)
2 | Invalid URL submitted
3 | Rate limit reached. Wait a second and try again
4 | IP-Address has been blocked because of violating our terms of service
5 | shrtcode code (slug) already taken/in use
6 | Unknown error
7 | No code specified ("code" parameter is empty)
8 | Invalid code submitted (code not found/there is no such short-link)
9 | Missing required parameters
10 | Trying to shorten a disallowed Link. More information on disallowed links
```
We can use that.